docs/examples.md

```diff
@@ -4,6 +4,21 @@ title: Examples
 ---
 
 > Add your own examples to our [Wiki Page](https://github.com/Codeception/CodeceptJS/wiki/Examples)
+## [TodoMVC Examples](https://github.com/codecept-js/examples)
+
+![](https://github.com/codecept-js/examples/raw/master/todo.png)
+
+Playground repository where you can run tests in different helpers on a basic single-page website.
+
+Tests repository demonstrate usage of
+
+* Puppeteer helper
+* WebDriver helper
+* TestCafe plugin
+* Toggle headless mode with env variables
+* PageObjects
+* Cucumber Syntax
```

Comment:
"syntax" instead of "Syntax"

Reply:
Taken from: https://github.com/Codeception/CodeceptJS/wiki/Examples
This page was generated from wiki